README.md

```diff
@@ -125,40 +102,22 @@ Here is a tutorial of the current development features:
 		   Some common rates are:  -r 29.97  -r 25  -r 30  -r 60
 
 
-#################
-    WEBSITES
-#################
+## Websites
 
-http://www.openshot.org/  (PROJECT WEBSITE and BLOG)
-http://launchpad.net/openshot/ (SOURCE CODE, BUG TRACKER, TRANSLATIONS)
-http://freshmeat.net/projects/openshot-video-editor/
-https://sourceforge.net/projects/openshotvideo/
-https://www.ohloh.net/p/openshot-video-editor/
+- http://www.openshot.org/  (Official website and blog)
+- http://launchpad.net/openshot/ (source code, bug tracker and translations)
+- http://freshmeat.net/projects/openshot-video-editor/
+- https://sourceforge.net/projects/openshotvideo/
+- https://www.ohloh.net/p/openshot-video-editor/
 
+## Report a bug
 
+You can add a bug here on GitHub but make sure to add plenty of detail about the issue and logs if possible.
```

Comment:
Perhaps we should link/reference [CONTRIBUTING.md](.github/CONTRIBUTING.md) here, for more detailed bug reporting instructions?

Reply:
Another great suggestion. I agree. Thanks!

Reply:
Submitted #1544 to relocate CONTRIBUTING.md.

Reply:
(Oh, and along the same lines as what I mentioned above about self-referential links, since this file will also — I assume — be included in the OpenShot source tarballs, I'd suggest avoiding language like "here on Github" that assumes the reader... well... is. Here. On Github. They may not be. :innocent: ) 

Perhaps something like "You can file an Issue in the OpenShot/openshot-qt project on Github..." with a link directly to https://github.com/OpenShot/openshot-qt/issues ? (I was first thinking https://github.com/OpenShot/openshot-qt/issues/new which will drop them right at the new-issue form, but really they should be encouraged to search for their issue before filing a duplicate report.)